### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by Abdelaziz-Mahdi**
Merging a project code to an empty repo to perform a code review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
```
Comment (Abdelaziz-Ali) [inline index.html:58]: Kindly use HTML5 semantic tags 🤓 (e.g., header, nav), A semantic element clearly describes its meaning to both the browser and the developer by defining the different parts of a web page like (article - aside - details - figcaption - figure -footer - headed - main - mark - nav - section - summary - time).
200432657-71a41001-fe59-47dc-974b-9830c0dd0822

You can find more information here 👉 [https://www.w3schools.com/html/html5_semantic_elements.asp](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url)

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Abdelaziz-Ali) [inline css/style.css:1]: You have successfully added a style sheet to your project 👌, please make sure to use CSS, CSS makes the front end of a website shine 😎 and it creates a great user experience, u can use it also to arrange the parts of the website.
More information here 👉 [https://www.devopsschool.com/blog/what-is-css-and-its-importance-in-web-development/](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url).

**Review by Abdelaziz-Mahdi — COMMENTED** [review]
.

File: index.html (hunk 1 above)
Comment (Abdelaziz-Mahdi) [inline index.html:58]: Kindly use HTML5 semantic tags 🤓 (e.g., header, nav), A semantic element clearly describes its meaning to both the browser and the developer by defining the different parts of a web page like (article - aside - details - figcaption - figure - footer - header - main - mark - nav - section - summary - time).

200432657-71a41001-fe59-47dc-974b-9830c0dd0822

You can find more information here 👉 [https://www.w3schools.com/html/html5_semantic_elements.asp](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url)

File: css/style.css (hunk 2 above)
Comment (Abdelaziz-Mahdi) [inline css/style.css:1]: You have successfully added a style sheet to your project 👌, please make sure to use CSS, CSS makes the front end of a website shine 😎 and it creates a great user experience, u can use it also to arrange the parts of the website.
More information here 👉 [https://www.devopsschool.com/blog/what-is-css-and-its-importance-in-web-development/](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url).

**Review by Abdelaziz-Mahdi — COMMENTED** [review]
# Changes Required ♻️ 

- Kindly review my comments and check what is required.
- Request another review after done from changes.

Thank you, and best of luck 🍀

**Review by Gambit142 — APPROVED** [review]
Hi @Abdelaziz-Ali 👋 ,

## Status: Approved 🟢   

#### Great work man. This is impressive work done 👏 😎 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Use of GitHub's Inline comments ✔️ 
- Comments were dished politely ✔️ 
- Changes requested were descriptive and very explanatory ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**